d2l-tspan.html

```diff
@@ -16,14 +16,37 @@
 				border: none;
 			}
 
-			#float {
+			.d2l-tspan-float,
+			:host([focused-styling]) .d2l-tspan-float:hover,
+			:host([_feedback-in-focus]) .d2l-tspan-float {
 				position: absolute;
 				width: 100%;
-				border: var(--d2l-table-border);
-				border-bottom: none;
 				padding: 1rem;
 				left: 0;
-				box-sizing: border-box;
+				box-sizing: border-box;			
+			}
+
+			.d2l-tspan-float {
+				border: var(--d2l-table-border);
+				z-index: 0;
+				border-bottom: none;
+				box-shadow: 0 1px 0 var(--d2l-table-border-color);
+			}
+
+			:host([focused-styling]) .d2l-tspan-float,
+			:host([focused-styling]) .d2l-tspan-float:hover,
+			:host([_feedback-in-focus]) .d2l-tspan-float {
```

Comment:
I think the `:host([_feedback-in-focus])` rule is unnecessary here since it will only be true if `focused-styling` is true, and the CSS already selects on that